Modules/rotatingtree.c

```diff
@@ -1,3 +1,9 @@
+#ifndef Py_BUILD_CORE_BUILTIN
+#  define Py_BUILD_CORE_MODULE 1
+#endif
```

Comment:
This is unfortunate.  We should avoid building non-builtin stdlib modules with `Py_BUILD_CORE_MODULE` or `Py_BUILD_CORE_BUILTIN`.  Is there anything we can do about that?  I suppose it would have to wait until `Py_Mutex` becomes public API.

Would you mind creating an issue about dropping this `#define` when that happens (and add a comment here referencing the issue)?  That way we don't forget about it. 😄

Reply:
Sure, I created a issue for this #116181

Reply:
Thanks!